components/com_joomgallery/views/userpanel/tmpl/default.php

```diff
@@ -152,11 +152,11 @@
                 <i class="icon-menu"></i>
               </span>
               <input type="text" style="display:none" name="order[]" size="5" value="<?php echo $item->ordering;?>" class="width-20 text-area-order " />
-            <?php else : ?>
+            <?php } else { ?>
               <span class="sortable-handler inactive" >
                 <i class="icon-menu"></i>
               </span>
-            <?php endif; ?>
+				<?php } ?>
```

Comment:
All the changes in this file seem to be unrelated to the new feature. I would suggest reverting these or why was this change necessary?